logical_switch_port.go

```diff
@@ -206,6 +207,28 @@ func (odbi *ovndb) lspSetOptionsImp(lsp string, options map[string]string) (*Ovn
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
+func (odbi *ovndb) lspSetDynamicAddressesImp(lsp string, address string) (*OvnCommand, error) {
+	row := make(OVNRow)
```

Comment:
Thanks for all the corrections. I think checking lsp is not null is still missed e.g. you can use chassis.go for reference https://github.com/eBay/go-ovn/blob/master/chassis.go#L38 where mandatory field name is always being checked before mutating ovn dbs. Kindly address corresponding UT for null LSP too?

Reply:
I have addressed the two places where I added new methods. But in the rest of the file, all the other existing functions like `lspAdd`, `lspDel`, `lspSetAddress` don't check for an empty string for the lsp. It makes sense to fail faster though than invoking the db transaction and failing.

Reply:
I can create a separate PR to address checking for empty strings for lsp name in other implementation methods for the LSP.